packages/e2e-tests/helpers/lexical.ts

```diff
@@ -70,5 +70,9 @@ export async function type(page: Page, selector: string, text: string, shouldSub
   if (shouldSubmit) {
     await delay(200);
     await submitCurrentText(page, selector);
+  } else {
+    // The typeahead popup sometimes sticks around and overlaps the save button.
+    // Ensure it goes away.
+    await hideTypeAheadSuggestions(page, selector);
```

Comment:
Wow! Could have sworn we had already added that but maybe I"m thinking of the screenshot tests.

Reply:
Looks like putting it here breaks two other logpoints tests.  Moving this to be up a level, right before we try to save.